src/bosh_azure_cpi/lib/cloud/azure/cloud.rb

```diff
@@ -14,21 +13,15 @@ class Cloud < Bosh::Cloud
     # Cloud initialization
     #
     # @param [Hash] options cloud options
-    def initialize(options)
-      options_dup = options.dup.freeze
-
-      @logger = Bosh::Clouds::Config.logger
-
-      @config = Bosh::AzureCloud::ConfigFactory.build(options_dup)
-
-      request_id = options_dup['azure']['request_id']
-      Bosh::AzureCloud::CPILogger.set_request_id(request_id) if request_id
+    def initialize
+      request_id = config.azure.request_id
```

Comment:
where is the code to initialize `Bosh::AzureCloud::Config`?

Reply:
added.

Reply:
added.